Created module for POI. POI intro, examples and Presentation Maker intro (#14)
apluslms/course-templates: m10_point_of_interest/02_examples.rst

Point of Interest - Examples
============================

> I will try to come up with useful examples that anyone can use as a starting point for creating their own content.

Reviewer: ```suggestion
This chapter provides some examples that anyone can use as a starting point for creating their own content.
```

> Point of Interest (POI) with all the options available

Reviewer: ```suggestion
Point of Interest with all the options available
```

------------------------------------------------------

> Underneath you can see POI directive with all the options. Normally it does not look like this, this is just for the reference.

Reviewer: How it should look like?. You must provide some details on how the directive actually looks.

Author: I renamed the title and changed this to just show all the options for reference.

> Study the point of interest and all its options. Later we will show more practical examples.

Reviewer: It is really necessary?

Author: I changed this to just show all the options for reference.

If there isn't next or previous then it's not required to make those options. And remember linking isn't mandatory.

Here is how you do it.

.. code-block:: none

    .. point-of-interest:: POI 1 title
      :id: first
      :next: second

      This is POI 1. It has link to the next POI with id of 2.

    .. point-of-interest:: POI 2 title
      :id: second
      :previous: first
      :next: third

      This POI is linked to the POI 1 with the previous option which has the id of the previous POI. In this case it is 1. Next works the same way.

    .. point-of-interest:: POI 3 title
      :id: third
      :previous: second

      This POI is linked to the POI 2.

**Result**

.. point-of-interest:: POI 1 title
  :id: first
  :next: second

  This is POI 1. It has link to the next POI with id of 2.

.. point-of-interest:: POI 2 title
  :id: second
  :previous: first
  :next: third

  This POI is linked to the POI 1 with the previous option which has the id of the previous POI. In this case it is 1. Next works the same way.

.. point-of-interest:: POI 3 title
  :id: third
  :previous: second

  This POI is linked to the POI 2.

> **Possible issues with linking**

Reviewer: Try adding an admonition here.

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

        .. column::
          :width: 1
          :column-class: text-center bg-info

          width of 1

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

        .. column::
          :width: 1
          :column-class: text-center bg-info

          width of 1

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

        .. column::
          :width: 1
          :column-class: text-center bg-info

          width of 1

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

        .. column::
          :width: 1
          :column-class: text-center bg-info

          width of 1

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

        .. column::
          :width: 1
          :column-class: text-center bg-info

          width of 1

        .. column::
          :width: 1
          :column-class: text-center

          width of 1

**Making columns with ::newcol (Deprecated)**

> This is deprecated and you should use row and column directives for creating columns and rows. With newcol you cannot create rows.

Reviewer: ```suggestion
the directive option `newcol` is deprecated and you should use row and column directives for creating columns and rows. 
```

.. code-block:: none

    .. point-of-interest:: Testing newcol
      :columns: 1 1 1

      This is on the first column. In this case columns option sets each column to be equal width. You could set it to any ratio like 3 1 1.

      ::newcol

      This is on the second column.

      ::newcol

      And this is on the third.

**Result**

.. point-of-interest:: Testing newcol
  :columns: 1 1 1

  This is on the first column. In this case columns option sets each column to be equal width. You could set it to any ratio like 3 1 1.

  ::newcol

  This is on the second column.

  ::newcol

  And this is on the third.

> I don't recommend mixing `newcol` with row and column directives. It will work but may not work the way you intend it to.

Reviewer: An admonition could highlight the importance of this.

.. code-block:: none

    :id: unique id, if not supplied a random id will be generated. Use alphanumeric IDs.
    :previous: id of previous point-of-interest (optional)
    :next: id of next point-of-interest (optional)
    :hidden: (if this flag is present, the content of this poi is hidden by default)
    :class: any additional css classes
    :height: fixed height in pixels
    :bgimg: path to background image
    :columns: used with newcol to set relative widths of poi content columns (e.e. 2 3 3) DEPRECATED
    :not_in_slides: a flag used with the presentation maker. This POI does not show in the slides if this is defined.
    :not_in_book: If this flag is given, this POI does not appear in the A+ content chapter.
    :no_poi_box: Removes surrounding box and navigation

Linking POI with another
------------------------

POI ids are used to link POI together. Each POI has next and previous links on the top of the POI to easily navigate between linked POI.

If you want to link POI 1, POI 2 and POI 3 you will need to set ids for the each POI and set next and previous options for the POI in order to get the links working.

> If there isn't next or previous then it's not required to make those options. And remember linking isn't mandatory.

Reviewer 2: `If there isn't next or previous`: this is bad English. You can rephrase it.
